USFMToolsSharp.Linter/USFMLinter.cs

```diff
@@ -16,14 +17,25 @@ public class USFMLinter
             new UnpairedEndMarkers(),
             new MissingTableRows()
         };
-        public List<LinterResult> Lint(USFMDocument input)
+        public async Task<List<LinterResult>> Lint(USFMDocument input)

```

Comment:
I would make this another separate method so that the public api doesn't change.

Reply:
@rbnswartz Updated :D